lib/msal-common/src/client/RefreshTokenClient.ts

```diff
@@ -63,7 +65,23 @@ export class RefreshTokenClient extends BaseClient {
             true,
             request.forceCache,
             requestId
-        );
+        ).then((result: AuthenticationResult) => {
+            atsMeasurement?.endMeasurement({
+                success: true,
+                refreshTokenSize: response.body.refresh_token?.length || 0
```

Comment:
I thought we had some discussion on cases where no refresh token was returned we'd not return a size of 0 but we wouldn't log that datapoint at all.

Reply:
@peterzenz Not logging in either case. 
From the testing point of view, the case where no refresh token is returned is semantically treated as same as having a token of length 0.